Networking/Networking/Requests/Request.swift

```diff
@@ -12,3 +12,9 @@ protocol Request: URLRequestConvertible {
     ///
     func responseDataValidator() -> ResponseDataValidator
 }
+
+extension URLRequest: Request {
```

Comment:
Nit: Consider adding a new file for `PlaceholderDataValidator` and moving this extension and the `PlaceholderDataValidator` declaration to a separate file.

Reply:
Good point - updated in fbb0e66c8accf3f43c2cbfaefe9e716671e8050d.